### DISCUSSION TITLE
Valorant

### DISCUSSION THREAD

**Discussion by syedfaizanqaiser** [cafali/SnapKey#100] [Compatibility List Submissions]
Got banned in valorant after a few days for using a 3rd party application and manipulating inputs.

**Comment by toinouH** [comment]
Was this application using the Windows API or injecting into the drivers of your input device ?

**Reply by syedfaizanqaiser** [reply]
The software claims to only use the windows API.Also I think that the riot anti cheat system got triggered due to my stats (high k/d and high headshot percentage ranging from 50-70).Also Im an immo player but I was playing on a plat account with my friends).I got reported every game by other players and even my random teammates.
Keeping in mind that I used my main account to play aswell with snapkey enabled and I didn't got banned on that account.So Its highly risky and I think that multiple reports caused the anticheat to flag this 3rd party software as a cheat.
I submitted a ticket aswell and the reply from support stated that I used a 3rd party application for manipulating inputs and that I violated the integrity of the game and that's why I got permanently banned.They haven't mentioned the software name ofcourse even upon requesting.
